reasoner_pydantic/base_model.py

```diff
@@ -27,6 +27,9 @@ def update(self, other):
             f"Model {self.__class__.__name__} has no update method"
         )
 
+    def get_field(self, field):
```

Comment:
What is this used for?

Reply:
I was originally going to use this method, but this is what we had discussed before, which provides a method to get an arbitrary field, if it exists for that model.